libnetdata/socket/security.c

```diff
@@ -176,6 +176,9 @@ void security_start_ssl(int selector) {
         }
```

Comment:
Did we have this before and verified it's not a problem? I don't remember...
![Screenshot from 2019-07-18 18-37-58](https://user-images.githubusercontent.com/43294513/61475589-73ceed00-a98b-11e9-9b45-2ea981cede58.png)

Reply:
Yes it is a false positive, case you use an "alt+f7" inside clion, you will see that the function security_start_ssl is always called with a valid value.